### ISSUE TITLE
.gcda is not generated

### ISSUE THREAD

**Issue by emartynov** [jonreid/XcodeCoverage#32]
I don't know if it issue of XcodeCoverage or Xcode itself. 

I have target with name 'app develop'. If I run test I don't see .gcda files generated. If I run tests for 'app' target everything works.

Except of renaming target are there solutions?

**Comment by jonreid** [comment]
So you have a target with a space in the name? Always fun with scripts. Try with the latest XcodeCoverage to see if it fixes things.

**Comment by emartynov** [comment]
Hi John,

I've tried the last one and it still has same issue. 
Is it something in XcodeCoverage scripts or Xcode instrumentations itself?

**Comment by emartynov** [comment]
Ok, after some investigation I found the issue.

I have two target in project ```app``` and ```app-develop```. 
Target ```app-develop``` produces ```app develop``` binary. 
Tests are pointed to ```app-develop```.

I added export environment variables to both targets looks like ```app``` was running last and replaced folder where coverage files were generated. As soon as I removed script running from this target it started working.

Issue might be closed.
